modules/ROOT/pages/tutorial-services.adoc

```diff
@@ -104,13 +104,64 @@ storage:
               /etc/issue.d/50_public-ipv4.issue
 ----
 
+TIP: You may be able to use local file references to systemd units instead of inlining them. See xref:#_using_butanes__files_dir_parameter_to_embed_files[Using butane's `--files-dir` Parameter to Embed Files] for more information.
+
 And then convert to Ignition:
 
 [source,bash]
 ----
 butane --pretty --strict services.bu --output services.ign
 ----
 
+=== Using butane's `--files-dir` Parameter to Embed Files
+
+Depending on the variant and version you use, you can use file references to systemd units instead of inlining them.
```

Comment:
One more:

```suggestion
Depending on the variant and version you use, you can use local file references to systemd units instead of inlining them.
```

Reply:
That should take care of all of them. Searched the project for the wrong substring and missed this one during a cursorily reread of the commit as well :(

Well spotted!